Consumer/src/main/java/com/finalboss/useCases/EventHandler.java

```diff
@@ -5,27 +5,32 @@
 import com.finalboss.domain.Operation;
 import com.finalboss.domain.YellowEvent;
 import com.finalboss.mapper.YellowEventMapper;
+import com.finalboss.repository.YellowEventRepository;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
 import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
-import java.util.logging.Logger;
+import org.slf4j.Logger;
 
+@Component
 public class EventHandler implements EventHandlerI {
 
-    private static final Logger log = Logger.getLogger(EventHandler.class.getName());
-    private final YellowEventRepositoryI yellowEventRepositoryI;
+    private static final Logger log = LoggerFactory.getLogger(EventHandler.class);
+    private final YellowEventRepository repo;
     private final YellowEventMapper yellowEventMapper;
 
-    public EventHandler(YellowEventRepositoryI yellowEventRepositoryI, YellowEventMapper yellowEventMapper) {
-        this.yellowEventRepositoryI = yellowEventRepositoryI;
+    public EventHandler(YellowEventRepository repo, YellowEventMapper yellowEventMapper) {
+        this.repo = repo;
         this.yellowEventMapper = yellowEventMapper;
     }
```

Comment:
please put an empty line here to separare the constructor's last curly brace from the `@Override`